### PR TITLE
fix delete method - task#318

### PR DESCRIPTION
**Pull request by SlakvichAleksandr**

### CODE REVIEW

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs [hunk 1]
```diff
@@ -272,7 +272,9 @@ public async Task DeleteAvailabilitySlot(long slotId)
         {
             var slot = await _context.AvailabilitySlots.FirstAsync(el => el.Id == slotId);
             var members = _context.SlotMembers.Where(member => member.SlotId == slotId);
+            var meetings = _context.Meetings.Where(_ => _.AvailabilitySlotId == slotId);
```
Comment (hekeemje) [inline backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs]: maybe make normal name in lambda instead of "_"

**Review by hekeemje — CHANGES_REQUESTED** [review]

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs (hunk 1 above)
Comment (SlakvichAleksandr) [reply backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs]: it doesn't matter :) it's only short construction

**Review by hekeemje — APPROVED** [review]
looks good

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs (hunk 1 above)
Comment (hekeemje) [reply backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs]: ok )

**Review by apofis96 — APPROVED** [review]

File: backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs (hunk 1 above)
Comment (tatianahutii) [reply backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs]: Not, it would better to use normal variable name

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**